src/inspector_io.cc

```diff
@@ -96,6 +97,12 @@ int CloseAsyncAndLoop(uv_async_t* async) {
   return uv_loop_close(async->loop);
 }
 
+void ReleasePairOnAsyncClose(uv_handle_t* async) {
+  AsyncAndAgent* pair = node::ContainerOf(&AsyncAndAgent::first,
+                                        reinterpret_cast<uv_async_t*>(async));
```

Comment:
I think this line is supposed to be indented to line up after the parentheses (that you have to do this after changing the name of the type in the line before is why I don't like this style, but its the node style)